src/main/openapi/ds-datahandler-openapi_v1.yaml

```diff
@@ -320,6 +318,26 @@ components:
                             [Preservica](https://preservica.com/) XIP content of the record
           example: 'direct'
           default: 'direct'
+        dateStampFormat:
+          type: string
+          enum:  ['day', 'date']
+          description: |
+            Special handling of record. Possible value are
+            * `day`:     Only use format yyyy-MM-dd as from/until parameters
+            * `date`:    Use UTC format yyyy-MM-ddThh:mm:ssZ as from/until parameters. This is default.                        
```

Comment:
`day` and `date` have the same meaning. Please change the enum to use words that signals what the difference is.

Reply:
date and datetime it is